frontend/src/lib/components/MapComponent.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
```

Comment:
This file needs to be formatted with prettier (I recommend the format on save setting so that you don't have to remember to manually format)